module/mailer/src/Domain/TestMail.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Ergonode\Mailer\Domain;
+
+use DateTime;
+use Ergonode\Core\Domain\ValueObject\Language;
+use Ergonode\SharedKernel\Domain\Collection\EmailCollection;
+use Ergonode\SharedKernel\Domain\ValueObject\Email;
+
+/**
+ * Test mail message
+ */
+class TestMail extends Mail
+{
+    /**
+     * @param Email    $to
+     * @param Language $language
+     */
+    public function __construct(Email $to, Language $language)
+    {
+        $this->createdAt = new DateTime();
```

Comment:
Perhaps use `parent::__constructor` - it'll make sure Mail is created appropriately right away with all params rather than failing in some weird scenario if Mail would be extended and this not.